### PR TITLE
fixed balance page stats

### PR DESCRIPTION
**Pull request by sdl433**
Note: Balance page loads slowly because it currently does arithmetic and clones lists for each date it traverses
Screenshot 2021-11-21 180930
.

### CODE REVIEW

File: back-end/app.js [hunk 1]
```diff
@@ -338,6 +337,21 @@ app.get("/api/get_transactions", async (request, response) => {
...
+  } catch (error) {
+    console.log("ERROR:");
+    console.log(error);
+    prettyPrintResponse(error);
+    return response.status(500).json({
+      err: error,
+    });
+  }
+})
+
```
Comment (tangym27) [inline back-end/app.js:354]: I don't think this does anything?

File: .idea/workspace.xml [hunk 2]
```diff
@@ -1,16 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (tangym27) [inline .idea/workspace.xml:1]: i think this whole page is unnecessary? if you're using jetbrains i think they force this on you unfortunately :(

**Review by tangym27 — APPROVED** [review]
Minor nitpicks but if you want to merge, feel free to. Nice job & thanks for fixing stats. I'm going to add Tomer to review it bc I did a quick glance through.

File: .idea/workspace.xml (hunk 2 above)
Comment (sdl433) [reply .idea/workspace.xml:1]: Haha yep, I recall putting this in my .gitignore, but it somehow managed to slip into a commit 😆

**Pull request merged**